### PR TITLE
power 264 connectome example

### PR DESCRIPTION
**Pull request by amadeuskanaan**
needs another look at edge thresholding

### CODE REVIEW

File: examples/connectivity/plot_power_connectome.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+
+from nilearn import datasets
+
+#fetch power atlas coords
+power = datasets.fetch_atlas_power_2011()
+power_coords = [(i[1], i[2], i[3]) for i in power.rois]
```
Comment (AlexandreAbraham) [inline examples/connectivity/plot_power_connectome.py]: This is equivalent to: `power_coords = power.rois[['x', 'y', 'z']]`.

File: examples/connectivity/plot_power_connectome.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+#fetch dataset
+adhd = datasets.fetch_adhd()
```
Comment (AlexandreAbraham) [inline examples/connectivity/plot_power_connectome.py]: `datasets.fetch_adhd(n_subjects=1)`

File: examples/connectivity/plot_power_connectome.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                                       detrend = True,
+                                       low_pass = 0.1,
+                                       high_pass = 0.01,
+                                       t_r = 2.5,
+                                      verbose = 5)
+
+timeseries = masker.fit_transform(adhd.func[0], confounds = adhd.confounds[0])
+
+# calculate connectivity and plot Power-264 correlation matrix
+connectivity = np.arctanh(np.corrcoef(timeseries.T))
```
Comment (AlexandreAbraham) [inline examples/connectivity/plot_power_connectome.py]: `np` is not defined.

File: examples/connectivity/plot_power_connectome.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+plt.imshow(connectivity, vmin = -1., vmax = 1.)
```
Comment (AlexandreAbraham) [inline examples/connectivity/plot_power_connectome.py]: `plt` is not defined.

File: examples/connectivity/plot_power_connectome.py [hunk 5]
```diff
@@ -0,0 +1,70 @@
+
+from nilearn import datasets
+
+#fetch power atlas coords
```
Comment (AlexandreAbraham) [inline examples/connectivity/plot_power_connectome.py]: Missing blank line for PEP8 compliance.

File: examples/connectivity/plot_power_connectome.py (hunk 4 above)
Comment (amadeuskanaan) [inline examples/connectivity/plot_power_connectome.py]: defined

File: examples/connectivity/plot_power_connectome.py (hunk 3 above)
Comment (amadeuskanaan) [inline examples/connectivity/plot_power_connectome.py]: defined

File: examples/connectivity/plot_power_connectome.py [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+import matplotlib.pyplot as plt
+from nilearn import datasets
+
+
+#fetch power atlas coords
+power = datasets.fetch_atlas_power_2011()
+power_coords = power.rois[['x', 'y', 'z']]
+
+#fetch dataset
+
```
Comment (GaelVaroquaux) [inline examples/connectivity/plot_power_connectome.py]: Could you remove this empty line.

File: examples/connectivity/plot_power_connectome.py [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+from nilearn import input_data
+masker = input_data.NiftiSpheresMasker(seeds = power_coords,
+                                       radius = 5.,
+                                       standardize = True,
+                                       smoothing_fwhm = 4,
+                                       detrend = True,
+                                       low_pass = 0.1,
+                                       high_pass = 0.01,
+                                       t_r = 2.5,
+                                      verbose = 5)
```
Comment (GaelVaroquaux) [inline examples/connectivity/plot_power_connectome.py]: Could you install a pep8 checker: this code does not satisfy pep8 standard (it has whitespace surrounding equal signs in the keyword arguments).

File: examples/connectivity/plot_power_connectome.py [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+connectivity = np.arctanh(np.corrcoef(timeseries.T))
+plt.imshow(connectivity, vmin = -1., vmax = 1.)
+plt.colorbar()
+plt.title('Power 264 Connectivity')
+
+# Plot the connectome
+from nilearn import plotting
+plotting.plot_connectome(connectivity,
+                         power_coords,
+                         edge_threshold ='99.8%',
```
Comment (GaelVaroquaux) [inline examples/connectivity/plot_power_connectome.py]: pep8 violation here also.

File: examples/connectivity/plot_power_connectome.py [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+                         power_coords,
+                         title='Covariance',
+                         edge_threshold ='99.8%',
+                         node_size = 20)
+
+# Display the sparse inverse covariance
+plt.figure(figsize=(5, 5))
+plt.imshow(-estimator.precision_, interpolation="nearest",
+           vmax=1, vmin=-1, cmap=plt.cm.RdBu_r)
+# And display the labels
```
Comment (GaelVaroquaux) [inline examples/connectivity/plot_power_connectome.py]: I would remove the line above.

**Comment by amadeuskanaan** [comment]
done

**Comment by lesteve** [comment]
The example doesn't work for me because power_coords is an array with a dtype=[('x', '<i8'), ('y', '<i8'), ('z', '<i8')] and shape (264,).

The definition for power_coords needs to be amended:

```python
power_coords = power.rois[['x', 'y', 'z']]
```

I believe you need to use a vstack as in examples/connectivity/plot_probabilistic_atlas_extraction.py

**Comment by amadeuskanaan** [comment]
i could use vstack or 

```python
power_coords = power.rois[['x', 'y', 'z']]
```
whats more pythonic ?

**Comment by amadeuskanaan** [comment]
There was a in week in copy/paste in last comment. See above now.

**Comment by lesteve** [comment]
vstack sounds good enough to me, not strong difference in pythonicity.

**Comment by AlexandreAbraham** [comment]
Thanks for your work, I'm going to take care of the last details in another PR.

File: examples/connectivity/plot_power_connectome.py [hunk 10]
```diff
@@ -0,0 +1,88 @@
+"""
+Extracting signals and plotting a connectome for the Power-264 seed-region atlas
```
Comment (banilo) [inline examples/connectivity/plot_power_connectome.py]: present progressive does not appear the most natural style choice at this point

File: examples/connectivity/plot_power_connectome.py [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+estimator.fit(timeseries)
+
+# Display the covariance
+plt.figure(figsize=(5, 5))
+plt.imshow(estimator.covariance_, interpolation="nearest",
+           vmax=1, vmin=-1, cmap=plt.cm.RdBu_r)
+plt.title('Covariance')
+
+# display the corresponding graph
+plotting.plot_connectome(estimator.covariance_,
```
Comment (banilo) [inline examples/connectivity/plot_power_connectome.py]: A minor thing but: beginners could be wondering why you are switching between ``plt`` and ``plotting``. Perhaps simply?

File: examples/connectivity/plot_power_connectome.py [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+                         power_coords,
+                         title='Covariance',
+                         edge_threshold='99.8%',
+                         node_size=20)
+
+# Display the sparse inverse covariance
```
Comment (banilo) [inline examples/connectivity/plot_power_connectome.py]: perhaps add: negated = partial correlations

File: examples/connectivity/plot_power_connectome.py (hunk 10 above)
Comment (AlexandreAbraham) [inline examples/connectivity/plot_power_connectome.py]: Fixed.

File: examples/connectivity/plot_power_connectome.py (hunk 11 above)
Comment (AlexandreAbraham) [inline examples/connectivity/plot_power_connectome.py]: plt is pyplot, plotting is nilearn. There is no simple way to handle it.

File: examples/connectivity/plot_power_connectome.py (hunk 12 above)
Comment (AlexandreAbraham) [inline examples/connectivity/plot_power_connectome.py]: This is not partial correlation but sparse inverse covariance.

File: examples/connectivity/plot_power_connectome.py [hunk 13]
```diff
@@ -0,0 +1,88 @@
+"""
+Extracting signals and plotting a connectome for the Power-264 seed-region atlas
+=============================================================
+
+This example shows how to extract signals from spherical seed-regions based
```
Comment (banilo) [inline examples/connectivity/plot_power_connectome.py]: what signals? "resting-state signals" could make this sentence more explict

File: examples/connectivity/plot_power_connectome.py [hunk 14]
```diff
@@ -0,0 +1,88 @@
...
+                                       detrend=True,
+                                       low_pass=0.1,
+                                       high_pass=0.01,
+                                       t_r=2.5,
+                                       verbose=5)
+
+timeseries = masker.fit_transform(adhd.func[0], confounds=adhd.confounds[0])
+
+# calculate connectivity and plot Power-264 correlation matrix
+connectivity = np.arctanh(np.corrcoef(timeseries.T))
```
Comment (banilo) [inline examples/connectivity/plot_power_connectome.py]: That is a lot of steps in one line for a tutorial example. In particular, perhaps do the ``arctanh`` in a separate line with a comment why you do this scaling.

**Pull request closed without merging**